samples/tls/logging.md

```diff
@@ -72,29 +79,40 @@ information about creating this secret manually.
 
 ## Specifying TLS for Ingress
 
-If you are using ingress and want to enable TLS on traffic into the cluster,
-you must manually create two Kubernetes secrets.
+If you are using ingress and want to enable TLS on communications into the cluster,
+you must manually create the following two Kubernetes secrets:
+
+- `elasticsearch-ingress-tls-secret`
+- `kibana-ingress-tls-secret`
 
-1. After you have obtained the certificates for the secrets, use this command
-to generate the secrets:
+**Note:** The process of generating the TLS certificates for these secrets is outside the scope for this example.
+
+1. After you have obtained the TLS certificates for each secret, use the following command to generate the secrets. You must run the command for each secret name.
 
 ```bash
-kubectl create secret tls $SECRET_NAME -n $NAMESPACE --key <cert-key> --cert <cert-file>
+kubectl create secret tls $SECRET_NAME -n $NAMESPACE --key $CERT_KEY --cert $CERT_FILE
 ```
 
-By default, the value of `namespace` that is used during the deployment process
-is `logging`. Run the command for for each of these values of `secret-name`:
+    where:
+
+- `$SECRET_NAME` is the one of the following values:
+
+  - The first time use `elasticsearch-ingress-tls-secret`.
+  - The second time use `kibana-ingress-tls-secret`.
+
+- `$NAMESPACE` is the value `logging` by default. If you deployed the logging components into a namespace with a different name, use that value instead.
+
+- `$CERT_KEY` is the TLS certificate key file associated with the secret name. Be sure to use the correct .key file in each instance of this command.
+
+- `$CERT_FILE` is the TLS certificate file associated with the secret name. Be sure to use the correct .crt file in each instance of this command.
 
-Use `elasticsearch-ingress-tls-secret` and `kibana-ingress-tls-secret` as
-values for `$SECRET_NAME`. Use the name of the namespace into which the logging
-components were deployed (such as `logging`) for the value of `$NAMESPACE`.
 
-2. In your local copy of the file
+1. In your local copy of the file
 `$USER_DIR\samples\tls\logging\user-values-elasticsearch-open.yaml`, update the
-values `kibana.logging.host.cluster.example.com` and
-`elasticsearch.logging.host.cluster.example.com` to reflect the namespace into
-which you have deployed the log monitoring components and the correct ingress
-host information.
+values in `kibana.logging.host.cluster.example.com` and
+`elasticsearch.logging.host.cluster.example.com` with the following information:
+   - the namespace into which you have deployed the logging and monitoring components 
+   - the correct ingress host information
 
```

Comment:
We may need to include an example to make this more clear.  Perhaps: _For example, if you have deployed the log monitoring components into the namespace "mymonitoring" and the ingress host is "myhost.example.com, you would use the values of kibana.mymonitoring.myhost.example.com and elasticsearch.mymonitoring.myhost.example.com._

Reply:
Agreed. That seems clearer.

Reply:
This is the revision:
In your local copy of the file $USER_DIR\logging\user-values-elasticsearch-open.yaml, update the following values:

- kibana.logging.host.cluster.example.com changes to the namespace into which you have deployed the logging and monitoring components
- elasticsearch.logging.host.cluster.example.com changes to the correct ingress host information

For example, if you deploy the log-monitoring components into the namespace mymonitoring and the ingress host is myhost.example.com, use the values of kibana.mymonitoring.myhost.example.com and elasticsearch.mymonitoring.myhost.example.com.

Reply:
Hmmm, both changes need to be made in both cases (e.g. Kibana and Elasticsearch).  By only including a single different value in the two bullet points, the reader may not understand that both changes are needed for both applications.  Perhaps:

In your local copy of the file $USER_DIR\logging\user-values-elasticsearch-open.yaml, update the values of `kibana.logging.host.cluster.example.com` and `elasticsearch.logging.host.cluster.example.com` as follows:

- change the namespace to the namespace into which you have deployed the log-monitoring components
 - change the host to the correct ingress host information

For example, if you deploy the log-monitoring components into the namespace `mymonitoring` and the ingress host is `myhost.example.com`, use the values of `kibana.mymonitoring.myhost.example.com` and `elasticsearch.mymonitoring.myhost.example.com`.

Reply:
To be clear, at a technical leve, the 'namespace' part of the hostname isn't **required** to match the actual logging namespace. It's just a clarifying convenience, not even a 'best practice.'

Reply:
Here is the revision:

In your local copy of the file
`$USER_DIR\logging\user-values-elasticsearch-open.yaml`, update each of the 
following values `kibana.logging.host.cluster.example.com` and `
elasticsearch.logging.host.cluster.example.com` in the following ways:

   - change the namespace to the namespace into which you have deployed the log-monitoring components
   - change the host to the correct ingress host information

    For example, if you deploy the log-monitoring components into the namespace 
    `mymonitoring` and the ingress host is `myhost.example.com`, use the values of 
    `kibana.mymonitoring.myhost.example.com` and 
    `elasticsearch.mymonitoring.myhost.example.com`.

Reply:
Responding to @BryanEllington : 
While it's not required on a technical level because its use is in the context of an example, I think the content is OK to stand without revision.